slm_lab/agent/memory/replay.py

```diff
@@ -269,11 +273,24 @@ def __init__(self, memory_spec, algorithm, body):
         self.state_buffer = deque(maxlen=self.stack_len)
         self.reset()
 
+    def reset(self):
```

Comment:
these 2 methods are inherited from super. when calling super reset the buffer append of zeros already happen. so they are not needed

Reply:
Yeah I see now. Thought that the state buffer got reloaded with states of size self.state_dim instead of self.raw_state_dim but was looking in the wrong place for where reset() and epi_reset() were inherited from.